packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,193 @@
+        <StackLayout gap={gap} direction={direction}>
+          {basicContact.metadata.map((metadata) => {
+            return (
+              <Text styleAs="label" key={metadata.label}>
+                {metadata.icon} {metadata.label} : {metadata.value}
+              </Text>
+            );
+          })}
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const Horizontal = () => {
+  return <ContactDetails avatarSize={2} gap={1} direction="row" />;
+};
+
+export const WithIcon = () => {
+  return <ContactDetails avatarSrc="" />;
+};
+
+export const CardEmbedded = () => {
+  return (
+    <Card>
+      <ContactDetails />
+    </Card>
+  );
+};
+
+export const QuickAction = () => {
+  return (
+    <StackLayout direction="row" gap={1}>
+      <Avatar
+        src="https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png"
+        aria-label="Woman Avatar"
+        fallbackIcon={<WomanIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3"> {basicContact.primary} </Text>
+        <Text styleAs="h4"> {basicContact.secondary}</Text>
+        <StackLayout gap={1} direction={"row"} separators>
+          <Tooltip content={"send email"}>
+            <Button variant="secondary">
+              <MessageIcon />
+            </Button>
+          </Tooltip>
+          <Tooltip content={"make call"}>
+            <Button variant="secondary">
+              <CallIcon />
+            </Button>
+          </Tooltip>
+          <Tooltip content={"send text"}>
+            <Button variant="secondary">
+              <ChatIcon />
+            </Button>
+          </Tooltip>
+        </StackLayout>
+      </StackLayout>
+    </StackLayout>
+  );
+};
+
+export const CollapsableDetails = () => {
+  const [toggleData, setToggleData] = useState(false);
+
+  function handleClick() {
+    setToggleData(!toggleData);
+  }
+  return (
+    <StackLayout direction="row" gap={1}>
+      <Avatar
+        src="https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png"
+        aria-label="Woman Avatar"
+        fallbackIcon={<WomanIcon />}
+        size={2}
+      />
+      <StackLayout direction={"column"} gap={1}>
+        <Text styleAs="h3"> {basicContact.primary} </Text>
+        <Text styleAs="h4"> {basicContact.secondary}</Text>
+        <StackLayout gap={1} direction={"row"}>
+          <Tooltip content={"send email"}>
+            <Button variant="secondary">
+              <MessageIcon />
+            </Button>
+          </Tooltip>
+          <Tooltip content={"make call"}>
+            <Button variant="secondary">
+              <CallIcon />
+            </Button>
+          </Tooltip>
+          <Tooltip content={"send text"}>
+            <Button variant="secondary">
+              <ChatIcon />
+            </Button>
+          </Tooltip>
+          <Button onClick={handleClick} variant="secondary">
```

Comment:
This isn't accessible, it needs to be labelled correctly.
- aria-hidden needs to be applied to the nested icons.
- An aria-label needs to be applied to the button
- The expanded state needs to be expressed etc.